tests/phpunit/test-wc-stripe.php

```diff
@@ -141,8 +141,6 @@ public function test_turning_on_upe_with_no_stripe_legacy_payment_methods_enable
 		$stripe_settings = WC_Stripe_Helper::get_stripe_settings();
 		$this->assertEquals( 'no', $stripe_settings['enabled'] );
 		$this->assertEquals( 'no', $stripe_settings['upe_checkout_experience_enabled'] );
-		$this->mock_payment_method_configurations( [], [] );
```

Comment:
Do we want to just delete this entire test since we don't update the PMC when enabling UPE?

Reply:
We are testing that when disabling the legacy checkout with no LPM currently enabled, no LPM will be automatically enabled for the new checkout.
I removed the PMC related check, but the `upe_checkout_experience_enabled` is still valid (for accounts that still need to re-authenticate)